### PR TITLE
Sort messages in berichtencentrum

### PR DESCRIPTION
**Pull request by claire-lovisa**

### CODE REVIEW

File: app/components/berichtencentrum/conversatie-view.js [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
 import { inject as service } from '@ember/service';
 import { tracked } from '@glimmer/tracking';
+import { sort } from '@ember/object/computed';
 
 export default class BerichtencentrumConversatieViewComponent extends Component {
   @service() router;
 
+  @sort('berichten', 'generalSort') sortedBerichten;
+
+  @tracked generalSort = Object.freeze(['verzonden']);
```
Comment (HugaertsDries) [inline app/components/berichtencentrum/conversatie-view.js:11]: What is this `Object.freeze` for?

**Review by HugaertsDries — APPROVED** [review]
Looks good, merge in to dev -> QA

File: app/components/berichtencentrum/conversatie-view.js (hunk 1 above)
Comment (claire-lovisa) [reply app/components/berichtencentrum/conversatie-view.js:11]: To ensure the sorting will only read the existing berichten and not modify then, not 100% it's required here now that you mention it

Comment (claire-lovisa) [reply app/components/berichtencentrum/conversatie-view.js:11]: Ah from what I read [here](https://stackoverflow.com/questions/55431665/ember-avoid-leaking-state-in-ember-objects-error-for-sorting-property) now that we have native classes it's not needed anymore so i'll just remove it :)

**Pull request merged**